tests/integration/test_dataapi.py

```diff
@@ -59,30 +52,100 @@ def _build_base_url(self):
 
         return base_url
 
-
     def test_real_request(self):
         """Make a request against Data Api to ensure the SDK works"""
         client = DataApi()
-        res = client.request(self._build_base_url() + endpoint, security, consumer_secret, request,
+
+        test_url = self._build_base_url() + items_endpoint
+
+        print("test_url is", test_url)
+
+        res = client.request(test_url, security, consumer_secret, items_request,
                              action)
-        print(res.request.url)
-        print(res.request.body)
-        print(res.content)
+
         returned_json = res.json()
+
         assert len(returned_json['data']) > 0
+
         returned_ref = returned_json['data'][0]['reference']
-        assert returned_ref in request['references']
+
+        assert returned_ref in items_request['references']
 
     def test_paging(self):
         """Verify that paging works"""
         client = DataApi()
-        pages = client.request_iter(self._build_base_url() + endpoint, security, consumer_secret,
-                                    request, action)
+
+        test_url = self._build_base_url() + items_endpoint
+
+        print("test_url is", test_url)
```

Comment:
Similar to above, we don't really want any print statements in here, and the extra blank lines should probably be reduced. Sometimes they're helpful for legibility, but every other line being blank is a bit too much.